docs/security.md

```diff
@@ -14,47 +14,85 @@ must authenticate to Mesos using a [DC/OS Service Account](https://docs.mesosphe
 
 Follow these instructions to [authenticate in strict mode](https://docs.mesosphere.com/services/spark/spark-auth/).
 
-# Using Mesos Secrets
+# Using the Secret Store
 
 DC/OS Enterprise allows users to add privileged information in the form of a file to the DC/OS secret store. These files
 can be referenced in Spark jobs and used for authentication and authorization with various external services (e.g.
 HDFS). For example, we use this functionality to pass the Kerberos Keytab. Details about how to use Secrets can be found
-at [official documentation](https://docs.mesosphere.com/latest/security/ent/secrets/). Once the secret has been added,
+at [official documentation](https://docs.mesosphere.com/latest/security/ent/secrets/).
+
+### Restricting Access to Secrets
+If only Spark requires access to a secret, store the secret in a path
+that matches the name of the Spark service (e.g. `spark/secret`).  See the [Secrets
+Documentation about Spaces][13] for more details about using spaces to restrict
+access to secrets.
+
+### Limitations
+Anyone who has access to the Spark service has access to all secrets
+available to Spark.
+Therefore, you should not allow anyone access to Spark who should not
+have access to those secrets.
+
+### Binary Secrets
+
+When you need to store binary files into DC/OS secrets store, for example a Kerberos keytab file, your file needs to be base64-encoded as specified in RFC 4648.
```

Comment:
This is not strictly necessary now. We could create a ticket to update this with the other docs. @mpereira dis some work on updating the section on binary secrets for the 1.11 release where we have native support.

Reply:
Good catch, there is already a ticket about updating this in the Spark docs (SPARK-620).